test/test_ppca.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pyro
+import pytest
+import torch
+from pyro import infer, optim
+
+from scvid.module import ProbabilisticPCAPyroModule
+
+n, p, k = 1000, 10, 2
+
+
+@pytest.fixture
+def x_np():
+    torch.manual_seed(1465)
+    z_nk = torch.randn(n, k)
+    w_kp = torch.randn(k, p)
+    sigma = 0.6
+    noise = sigma * torch.randn(n, p)
+    x_np = z_nk @ w_kp + noise
+    return x_np
+
+
+@pytest.mark.parametrize(
+    "marginalize", [False, True], ids=["unmarginalized", "marginalized"]
+)
+@pytest.mark.parametrize("minibatch", [False, True], ids=["fullbatch", "minibatch"])
+def test_probabilistic_pca(x_np, minibatch, marginalize):
+    x_mean = x_np.mean(axis=0)
+
+    pyro.clear_param_store()
+    ppca = ProbabilisticPCAPyroModule(
+        n_obs=n, n_vars=p, n_components=k, mean=x_mean, marginalize_z=marginalize
+    )
+    elbo = infer.Trace_ELBO()
+    adam = optim.Adam({"lr": 1e-2})
```

Comment:
Good test!

(unrelated to this PR -- but I have a feeling that scheduling the LR e.g. w/ the usual cosine annealing might do wonders with PPCA. why? because it's a simple model w/ a convex loss and after the initial convergence to a rough solution, annealing can let the "ball" roll down to the global minimum. It'd be great to try this out in your experiments!).